WooCommerce/WooCommerceTests/UnitTests.xctestplan

```diff
@@ -31,6 +31,8 @@
     {
       "skippedTests" : [
         "CardReaderConnectionControllerTests",
+        "InAppPurchaseStoreTests\/test_iap_supported_in_canada()",
+        "InAppPurchaseStoreTests\/test_load_products_fails_if_iap_unsupported()",
```

Comment:
Have you considered using `XCTSkip` to skip the tests? 

Not a big deal, but I personally prefer that approach because:

- It's closer to the actual test being skipped, which makes it easier to notice if you are looking at the test code, something that happens more often than looking at the test plan GUI view.
- It logs information about the test having been skipped in the console and test report, maintaining awareness about it

Reply:
A more complex alternative would be to use `XCTExpectFailure`, more info pbzQyC-423-p2#comment-4208

The value there would be that if something changes in a new version of Xcode so that the tests can now pass, the fact that they pass would trigger a failure (expected to fail, but passed) allowing us to updated the tests accordingly.

Reply:
Thanks. Yes, it makes sense. Having it in the code leaves it less obscure. Checking skipped tests within a test plan is likely not a part of developers' workflow.